enginetest/queries/index_query_plans.go

```diff
@@ -5919,4 +5919,20 @@ var IndexPlanTests = []QueryPlanTest{
 			" └─ columns: [pk v1 v2 v3 v4]\n" +
 			"",
 	},
+	{
+		Query: "select * from comp_index_t3 where v = 'a'",
+		ExpectedPlan: "Filter(comp_index_t3.v = 'a')\n" +
+			" └─ IndexedTableAccess(comp_index_t3)\n" +
+			"     ├─ index: [comp_index_t3.v]\n" +
+			"     ├─ filters: [{[a, a]}]\n" +
+			"     └─ columns: [pk v]\n",
+	},
+	{
+		Query: "select * from comp_index_t3 where v like 'a%'",
```

Comment:
might be worth adding two plans + correctness tests where the simplifications are applied but index isn't, like `select * From mytable where s like 'first%'` and `select * from mytable where s like 'first row'`